tests/schemas/input/test_json.py

```diff
@@ -11,7 +11,7 @@
 """
 
 root = pathlib.Path(__file__).parent.parent
-json_paths = list(root.glob('*.json'))
+json_paths = list(root.glob('schemas/input/*.json'))
```

Comment:
@dhimmel - I'm doing something wrong here. Trying to adjust the paths for the updated repo structure in this PR.

I'm assuming I actually need to remove this commit, and instead adjust the root, but I'm not sure how.

Can you help please?

Reply:
Looking through https://github.com/citation-style-language/schema/pull/212/checks?check_run_id=724834161#step:5:53

```
E       FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/schema/schema/tests/schemas/csl-citation.json'
```

I think you probably need to add the new paths in :

https://github.com/citation-style-language/schema/blob/2bbf44f8eb51f9ed5fa2cc84f319d8e4833a4174/tests/test_json.py#L25-L28

Reply:
`json_paths` is used by

https://github.com/citation-style-language/schema/blob/2bbf44f8eb51f9ed5fa2cc84f319d8e4833a4174/tests/test_json.py#L52-L57

Just tests whether json files in repo are parseable as JSON. So I might change this to:

```suggestion
json_paths = list(root.glob('**/*.json'))
```

Will find any `.json` file in the repo.

Reply:
So is the test failing now because the test code is actually in master?  E.g. this code is right now, but it just cannot avoid conflicting with master?

Reply:
> So is the test failing now because the test code is actually in master?

No I think it's failing because of the issue in https://github.com/citation-style-language/schema/pull/212#discussion_r432952845